packages/frontend/src/pages/Playground.svelte

```diff
@@ -1,5 +1,151 @@
+    .then(() => {
+      errorMsg = '';
+      sendEnabled = false;
+      prompt = '';
+    })
+    .catch((err: unknown) => {
+      errorMsg = String(err);
+      sendEnabled = true;
+    });
+}
+
+afterUpdate(() => {
+  if (!conversation?.messages.length) {
+    return;
+  }
+  const latest = conversation.messages[conversation.messages.length - 1];
+  if (isUserChat(latest) || (isAssistantChat(latest) && isPendingChat(latest))) {
+    scrollToBottom(scrollable);
+  }
+});
+
+async function scrollToBottom(element: Element) {
+  element.scroll?.({ top: element.scrollHeight, behavior: 'smooth' });
+}
+
+function elapsedTime(msg: AssistantChat): string {
+  if (isPendingChat(msg)) {
+    return ((Date.now() - msg.timestamp) / 1000).toFixed(1);
+  } else if (!!msg.completed) {
+    return ((msg.completed - msg.timestamp) / 1000).toFixed(1);
+  } else {
+    // should not happen
+    return '';
+  }
+}
 </script>
 
-{playgroundId}
+{#if playground}
+  <NavPage title="{playground?.name}" searchEnabled="{false}">
+    <svelte:fragment slot="subtitle">{model?.name}</svelte:fragment>
+    <svelte:fragment slot="content">
+      <div class="flex flex-col w-full h-full">
+        <div bind:this="{scrollable}" aria-label="conversation" class="w-full h-full overflow-auto">
+          {#if conversation?.messages}
+            <ul class="p-4">
+              {#each conversation?.messages as message}
+                <li class="m-4">
+                  <div class="text-lg" class:text-right="{isAssistantChat(message)}">{roleNames[message.role]}</div>
+                  <div
+                    class="p-4 rounded-md"
+                    class:bg-charcoal-400="{isUserChat(message)}"
+                    class:bg-charcoal-900="{isAssistantChat(message)}"
+                    class:ml-8="{isAssistantChat(message)}"
+                    class:mr-8="{isUserChat(message)}">
+                    {#each getMessageParagraphs(message) as paragraph}
+                      <p>{paragraph}</p>
+                    {/each}
+                  </div>
+                  {#if isAssistantChat(message)}
+                    <div class="text-sm text-gray-400 text-right" aria-label="elapsed">{elapsedTime(message)} s</div>
+                  {/if}
+                  <div></div>
+                </li>
+              {/each}
+            </ul>
+          {/if}
+        </div>
+        {#if errorMsg}
+          <div class="text-red-500 text-sm p-2">{errorMsg}</div>
+        {/if}
+        <div class="flex flex-row flex-none w-full">
+          <textarea
+            aria-label="prompt"
+            bind:value="{prompt}"
+            rows="2"
+            class="w-full p-2 outline-none text-sm bg-charcoal-800 rounded-sm text-gray-700 placeholder-gray-700"
+            placeholder="Type your prompt here"></textarea>
+
+          <div class="flex-none text-right m-4">
+            <Button disabled="{!sendEnabled}" on:click="{() => askPlayground()}">Send prompt</Button>
```

Comment:
Please a title saying why it is disabled.
<img width="788" alt="image" src="https://github.com/projectatomic/ai-studio/assets/42176370/f0a4dfc2-de9a-4287-878a-fc35eb3a070e">

As a user I feel frustated when I cannot click on a button and I don't know why